models/players.js

```diff
@@ -135,6 +144,19 @@ formatEUSoccerPlayers = function(response){
   return players;
 }
 
+formatMLBPlayers = function(response, team_id){
+  parseString(response, function (err, result) {
+    var str = result[Object.keys(result)[0]];
+        for (j=0; j < str.team.length; j++){ 
+          if (str.team[j].$.id == team_id){
```

Comment:
No, this URL is actually set up with all teams listed.
[roster
  [teams
    <team Id 1 players <1> <2>..>
    <team Id 2 players <1> <2>..>
    <team Id 3 players <1> <2>..>
  ]
]

Kelsey 
Edmodo Customer Success

> On Feb 11, 2015, at 9:55 PM, Blake Parkinson <notifications@github.com> wrote:
> 
> In models/players.js:
> 
> > @@ -135,6 +144,19 @@ formatEUSoccerPlayers = function(response){
> >    return players;
> >  }
> >  
> > +formatMLBPlayers = function(response, team_id){
> > +  parseString(response, function (err, result) {
> > +    var str = result[Object.keys(result)[0]];
> > +        for (j=0; j < str.team.length; j++){ 
> > +          if (str.team[j].$.id == team_id){
> is this just to be extra cautious? If so, maybe add an else clause outputting a message?
> 
> —
> Reply to this email directly or view it on GitHub.
>